### ISSUE TITLE
release schedule

### ISSUE THREAD

**Issue by whoDoneItAgain** [aws-cloudformation/cfn-lint#3003]
is there a defined release schedule? 

the fix for #2995 was merged last week but has not been added to a release yet. To get around the bug I am building locally.

I'm fine with my workaround but made me curious if there was an offical release schedule.

Thanks

**Comment by kddejong** [comment]
It should be today.  I try to release weekly when the specs are released but it came out a little late last week.

**Comment by kddejong** [comment]
https://pypi.org/project/cfn-lint/0.83.8/

**Comment by PatMyron** [comment]
Yeah released somewhere around ~weekly from looking at https://pypi.org/project/cfn-lint/#history:

**Releases by year:**
2019: 57
2020: 44
2021: 32
2022: 42
2023: 57 + 11 `1.x`
